### PR TITLE
add seed backup page

### PR DESCRIPTION
**Pull request by oshorefueled**
#### Resolves

Issue #144 

#### What's new

This PR adds the seed backup page to the app. 

#### Relevant screenshots or logs
Backup button on wallets page
Screenshot 2020-06-25 at 9 46 12 AM

Screenshot 2020-06-24 at 10 42 42 PM

Screenshot 2020-06-24 at 10 42 53 PM

Screenshot 2020-06-24 at 10 43 22 PM

Screenshot 2020-06-24 at 10 52 05 PM

Screenshot 2020-06-24 at 10 52 23 PM

### CODE REVIEW

File: ui/seed_backup_page.go [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+	}
+}
+
+func (pg *backupPage) clearError() {
+	go func() {
+		time.AfterFunc(time.Second*3, func() {
+			pg.error = ""
+		})
+	}()
+}
```
Comment (Sirmorrison) [inline ui/seed_backup_page.go]: time.AfterFunc() runs within a go routine by default. and you are currently using it with another go routine making it 2 go routines per call. why so?

**Review by Sirmorrison — CHANGES_REQUESTED** [review]

**Comment by Sirmorrison** [comment]
I noticed that if i am on the **Verify seed phrase** page (2/2) and i click the back button it sends me straight to the wallet home page. 

Is there a way i could use the back button to navigate to the previous page rather than the home page. Reason is, what if the user wants to verify something on the page (1/2) he will have to start the process all over again.

File: ui/seed_backup_page.go [hunk 2]
```diff
@@ -0,0 +1,538 @@
...
+			pg.seedPhrase = strings.Split(s, " ")
+			pg.populateSuggestionSeeds()
+			pg.active++
+		case verifyView:
+			if !checkSlice(pg.selectedSeeds) {
+				return
+			}
+			s := strings.Join(pg.selectedSeeds, " ")
+			if !dcrlibwallet.VerifySeed(s) {
+				pg.error = "Failed to verify. Please go through every word and try again."
```
Comment (Sirmorrison) [inline ui/seed_backup_page.go]: The error message here and on line 515 are repeating. Kind initialize a variable for this.

File: ui/seed_backup_page.go [hunk 3]
```diff
@@ -0,0 +1,538 @@
...
+func (pg *backupPage) layout() {
+	pg.theme.Surface(pg.gtx, func() {
+		toMax(pg.gtx)
+		layout.Flex{Axis: layout.Vertical, Alignment: layout.Start}.Layout(pg.gtx,
+			layout.Rigid(func() {
+				pg.action.Background = pg.theme.Color.Hint
+				switch pg.active {
+				case infoView:
+					if pg.verifyCheckBoxes() {
+						pg.action.Background = pg.theme.Color.Primary
```
Comment (Sirmorrison) [inline ui/seed_backup_page.go]: `pg.action.Background = pg.theme.Color.Primary` is repeated about 3 times sames as `pg.action.Color = pg.theme.Color.InvText`

These 2 can be added to `updateActionBackground()` and `updateActionColor()` respectively and called where needed.

**Review by Sirmorrison — CHANGES_REQUESTED** [review]

**Comment by oshorefueled** [comment]
> I noticed that if i am on the **Verify seed phrase** page (2/2) and i click the back button it sends me straight to the wallet home page.
> 
> Is there a way i could use the back button to navigate to the previous page rather than the home page. Reason is, what if the user wants to verify something on the page (1/2) he will have to start the process all over again.

That's the same behaviour on the mobile.

**Review by Sirmorrison — APPROVED** [review]
PR looks good to go.

**Review by beansgum — APPROVED** [review]

**Pull request merged**